docs/contracts.rst

```diff
@@ -24,8 +24,8 @@ Creating contracts programmatically on Ethereum is best done via using the JavaS
 As of today it has a method called `web3.eth.Contract <https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#new-contract>`_
 to facilitate contract creation.
 
-When a contract is created, its constructor (a function declared with the
-``constructor`` keyword) is executed once.
+When a contract is created, its constructor (a function declared with the ``constructor`` keyword) is executed once, including any code in the constructor.
```

Comment:
I wonder if we need to explicitly state that when executing the constructor we are actually executing any code in the constructor? Or maybe I am reading it wrongly.

Reply:
@axic Took another pass.